milli/src/search/facet/filter.rs

```diff
@@ -398,16 +408,21 @@ impl<'a> Filter<'a> {
                         rtxn,
                         index,
                         filterable_fields,
+                        universe,
                     )?;
                     for f in subfilters_iter {
                         if bitmap.is_empty() {
                             return Ok(bitmap);
                         }
+                        // TODO We are doing the intersections two times,
+                        //      it could be more efficient
+                        //      Can't I just replace this `&=` by an `=`?
                         bitmap &= Self::inner_evaluate(
```

Comment:
Pretty sure this is simply an `=` here. All the intersections were already made. I prefer to keep this extra `&=` in case we forgot an intersection when deserializing somewhere.